inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/transform/TransformFunctionDocServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.service.transform;
+
+import org.apache.inlong.manager.pojo.common.PageResult;
+import org.apache.inlong.manager.pojo.transform.TransformFunctionDocRequest;
+import org.apache.inlong.manager.pojo.transform.TransformFunctionDocResponse;
+import org.apache.inlong.sdk.transform.process.function.FunctionTools;
+import org.apache.inlong.sdk.transform.process.pojo.FunctionInfo;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Service
+public class TransformFunctionDocServiceImpl implements TransformFunctionDocService {
+
+    private final Map<String, Set<FunctionInfo>> functionDocMap = FunctionTools.getFunctionDoc();
+
+    @Override
+    public PageResult<TransformFunctionDocResponse> getFunctionDocs(TransformFunctionDocRequest request) {
+        String type = request.getType();
+        String name = request.getName();
+        int pageNum = request.getPageNum();
+        int pageSize = request.getPageSize();
+
```

Comment:
Please add some necessary logs. For example, printing the parameters of a request.

Reply:
done